vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/ICookieManager.java

```diff
@@ -26,8 +25,6 @@
 {
     void addCookie(String cookieName, String cookieValue, String path, String urlAsString);
 
-    void addCookie(String cookieName, String cookieValue, String path, URI uri);
```

Comment:
this should go in a separate PR

Reply:
https://github.com/vividus-framework/vividus/commit/35ae64da1162bf5200592ee20610e2a013d8965f